doc/docs.md

```diff
@@ -6054,7 +6054,7 @@ Full list of builtin options:
 | `solaris`, `termux`            |                  |                               | `js_node`, `js_browser`, `js_freestanding`    |
 | `serenity`, `vinix`, `plan9`   |                  |                               | `interpreter`, `es5`, `profile`, `wasm32`     |
 |                                |                  |                               | `wasm32_emscripten`, `wasm32_wasi`            |
-|                                |                  |                               | `native`, `autofree`                          |
+|                                |                  |                               | `native`, `autofree`, `compress`              |
```

Comment:
`compress` is not a builtin option name, i.e. you can not do `$if compress {` .
Imho revert this change.

Reply:
done